tests/functional/harris/harris_3d.py

```
@@ -0,0 +1,172 @@
+        tag_buffer=3,
+    )
+
+    def density(x, y, z):
+        Ly = sim.simulation_domain()[1]
+        return (
+            0.4
+            + 1.0 / np.cosh((y - Ly * 0.3) / L) ** 2
+            + 1.0 / np.cosh((y - Ly * 0.7) / L) ** 2
+        )
+
+    def S(y, y0, l):
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)